### PR TITLE
add additional reference to wiki article for MAD

### PR DESCRIPTION
**Pull request by davidecarlson**
This PR adds an additional reference to the wiki article on MAD. Note that I did not replace the existing link, but added a second one to point to an article that users can read to understand more about MAD since the existing reference redirects to [this page](https://qiagen.my.salesforce-sites.com/KnowledgeBase/KnowledgeNavigatorPage?id=kA4Sc0000000QLtKAM&categoryName=OmicSoft_Suite,OmicSoft_Lands), and it's hard to find actual info regarding MAD at that page.

## PR checklist

- [X] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/differentialabundance/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/differentialabundance _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [X] Ensure the test suite passes (`nf-test test main.nf.test -profile test,docker`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by WackerO** [comment]
Hey there, I just checked the original link. It is actually saved on archive.org:
https://web.archive.org/web/20221227023505/https://wiki.arrayserver.com/wiki/index.php?title=CorrelationQC.pdf
The images do not load, but the section about calculating the MAD score is displayed just fine.

I also searched on the arrayserver website and found this page:
https://qiagen.my.salesforce-sites.com/KnowledgeBase/KnowledgeNavigatorPage?id=kA4Sc0000000YJRKA2&categoryName=OmicSoft_Suite%2COmicSoft_Lands
I think that should be the current version of the previous link. I wonder if it makes sense to save the new link to archive.org and then use the archived version in the pipeline docs? (Unfortunately, the "Save Page Now" button on https://web.archive.org is blocked right now.)

**Comment by davidecarlson** [comment]
That plan is fine with me, @WackerO.  Hopefully the "Save Page Now" option will be back soon.

File: assets/differentialabundance_report.Rmd [hunk 1]
```diff
@@ -726,7 +726,7 @@ iv_min_group_sizes <- unlist(lapply(informative_variables, function(x) min(table
 
 if (any(iv_min_group_sizes > 2)){
     cat("\n### Outlier detection {.tabset}\n")
-    cat("\nOutlier detection based on [median absolute deviation](https://wiki.arrayserver.com/wiki/index.php?title=CorrelationQC.pdf) was undertaken, the outlier scoring is plotted below.\n")
+    cat("\nOutlier detection based on [median absolute deviation](https://wiki.arrayserver.com/wiki/index.php?title=CorrelationQC.pdf) was undertaken, the outlier scoring is plotted below. For more on MAD, see [this wiki article](https://en.wikipedia.org/wiki/Median_absolute_deviation).\n")
```
Comment (WackerO) [inline assets/differentialabundance_report.Rmd]:
```suggestion
    cat("\nOutlier detection based on [median absolute deviation](https://archive.ph/o3thZ) was undertaken, the outlier scoring is plotted below. For more on MAD, see [this wiki article](https://en.wikipedia.org/wiki/Median_absolute_deviation).\n")
```

File: nextflow_schema.json [hunk 2]
```diff
@@ -373,7 +373,7 @@
                 "exploratory_mad_threshold": {
                     "type": "integer",
                     "default": -5,
-                    "help_text": "MAD = median absolute deviation. A threshold on this value is used to define observations (samples) as outliers, or not, in exploratory plots. Based on the definition at https://wiki.arrayserver.com/wiki/index.php?title=CorrelationQC.pdf. ",
+                    "help_text": "MAD = median absolute deviation. A threshold on this value is used to define observations (samples) as outliers, or not, in exploratory plots. Based on the definition at https://wiki.arrayserver.com/wiki/index.php?title=CorrelationQC.pdf. For more on MAD, see https://en.wikipedia.org/wiki/Median_absolute_deviation.",
```
Comment (WackerO) [inline nextflow_schema.json]:
```suggestion
                    "help_text": "MAD = median absolute deviation. A threshold on this value is used to define observations (samples) as outliers, or not, in exploratory plots. Based on the definition at https://archive.ph/o3thZ. For more on MAD, see https://en.wikipedia.org/wiki/Median_absolute_deviation.",
```

**Review by WackerO — APPROVED** [review]

**Pull request merged**